pkg/osutil/tar_test.go

```diff
@@ -7,31 +7,20 @@ import (
 	"archive/tar"
 	"bytes"
 	"io"
-	"path/filepath"
 	"testing"
 
 	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
 )
 
 func TestTarDirectory(t *testing.T) {
 	dir := t.TempDir()
-
 	items := map[string]string{
 		"file1.txt":     "first file content",
 		"dir/file2.txt": "second file content",
 		"empty.txt":     "",
 	}
-
-	for path, content := range items {
-		fullPath := filepath.Join(dir, path)
-		dirPath := filepath.Dir(fullPath)
-		if err := MkdirAll(dirPath); err != nil {
-			t.Fatalf("mkdir %q failed: %v", dirPath, err)
-		}
-		if err := WriteFile(fullPath, []byte(content)); err != nil {
-			t.Fatalf("write file failed: %v", err)
-		}
-	}
+	require.NoError(t, FillDirectory(dir, items))
```

Comment:
`Package require implements the same assertions as the `assert` package but stops test execution when a test fails.`

Why do you think it is better? Should we use it wider?